frontend-react/src/pages/support/legacy-page-content/ServiceRequest.tsx

```diff
@@ -0,0 +1,25 @@
+import { Helmet } from "react-helmet";
+import DOMPurify from "dompurify";
+import { Alert } from "@trussworks/react-uswds";
+
+import site from "../../../content/site.json";
+
+export const ServiceRequest = () => {
+    return (
```

Comment:
Without this embedding the form as an `iframe`, you could actually make this a Markdown file. I figured the original intent was because `iframe` didn't work with our html-in-md extension, but all that's left is a link on the page.